tools/copy_data.py

```diff
@@ -0,0 +1,166 @@
+import psycopg2
+from config import config
+
+#A function that create table in db
+def create_table():
+    command = (
+        """
+        CREATE TABLE gps5 (
+            linia_id INT,
+            kurs_id INT,
+            pojazd_id INT,
+            przystanek_id INT,
+            poprzedni_przystanek_id INT,
+            data_czas_zdarzenia timestamp,
+            dlugosc_geograficzna varchar,
+            szerokosc_geograficzna varchar,
+            nazwa_linii varchar,
+            numer_kursu varchar,
+            przystanek_poczatkowy varchar,
+            przystanek_koncowy varchar,
+            nazwa_przystanku varchar,
+            numer_boczny varchar,
+            numer_pojazdu varchar,
+            poprzedni_przystanek varchar
+        );
+        """)
+    conn = None
+    try:
+        # read the connection parameters
+        params = config()
+        # connect to the PostgreSQL server
+        conn = psycopg2.connect(**params)
+        cur = conn.cursor()
+        # create table 
+        cur.execute(command)
+        # close communication with the PostgreSQL database server
+        cur.close()
+        # commit the changes
+        conn.commit()
+        print("Table created")
+    except (Exception, psycopg2.DatabaseError) as error:
+        print(error)
+    finally:
+        if conn is not None:
+            conn.close()
+
+#A function that insert data from csv to table in db
```

Comment:
Jeśli ten komentarz nie niesie ze sobą wartości to nie warto go utrzymywać. Z resztą, komentarze z reguły są zbędnę. Jak to się mówi, kod powinien być self explanatory.